server/feeders/restaurantfeeder.js

```diff
@@ -85,15 +85,15 @@ var feedOrionRestaurants = function() {
           utils.sendRequest('POST', attributes, null, fwHeaders)
```

Comment:
(Lines 8-12) Need full stop at the end of line
(Line 29) nit number --> Number
(Line 61) Commented code